### ISSUE TITLE
Newbie docs / missing something?

### ISSUE THREAD

**Issue by azaroth42** [sul-dlss/triannon#172]
For fun (actually to see if I could load annotations from newspapers), I grabbed the code to run locally.

Following the docs I got to:
[azaroth@localhost triannon]$ rake triannon:create_root_container
rake aborted!
LoadError: cannot load such file -- rdoc/task
/home/azaroth/Development/triannon/triannon/Rakefile:105:in `require'
/home/azaroth/Development/triannon/triannon/Rakefile:105:in `<top (required)>'

And same error message with just rake

Missing dependency? Missing file in the repo?

**Comment by azaroth42** [comment]
Docs should also list the need for a JS runtime with link to NPM.  
I found the install docs at the top of:
   https://github.com/iiif/m1

To be useful. (Thanks @aeschylus)

**Comment by cbeer** [comment]
> Docs should also list the need for a JS runtime with link to NPM.

That's a Rails requirement and none of our business.

**Comment by cbeer** [comment]
> [azaroth@localhost triannon]$ rake triannon:create_root_container

Did you try `bundle exec rake` (and, did you `bundle install` first?)

**Comment by azaroth42** [comment]
I did wonder why a ruby app needed a JS runtime... but it prevents the tests from running if you don't have it. Perhaps we (dlss) can have a newbie doc that repos can point to?

I did bundle install yes :) And then just rake / rake triannon:create_root_container, as per README... though following the rails version at the top rather than the test instance at the bottom.  Now that I have an internal triannon.yml it at least gets past the (cryptic to me) rdoc/task error.

Guess I should pair and take notes instead of messing around at 8am on a Saturday!

**Comment by cbeer** [comment]
Ah, I didn't know you were trying to run these from the gem, not generating a new application. That said, I can't figure out why we're keeping (broken) rdoc tasks around, so: https://github.com/sul-dlss/triannon/pull/173

**Comment by azaroth42** [comment]
Because I was walking through the README from top to bottom, being impatient :)
Now stuck with jetty not listening on 8983 after doing the install rake tasks and then jetty start :(  Will debug with @ndushay next week

**Comment by ndushay** [comment]
A ha!  I know the problem.  After running 

```
rake engine_cart:generate
```

you can either do:

```
cd spec/internal
rake triannon:create_root_container
```

OR

```
rake triannon:console
pry(main)> Triannon::RootAnnotationCreator.create
pry(main)> exit
```

I will modify the README.

**Comment by ndushay** [comment]
hah - it IS documented, under "Running the application in development"

**Comment by ndushay** [comment]
Have revised README in #173, and have added information on how to check if Solr and Fedora4 are running ok in jetty and what to do if they aren't.

**Comment by ndushay** [comment]
@azaroth42 if you feel I have adequately addressed the confusion, please close this issue.

**Comment by azaroth42** [comment]
Yep, looks great :) Will try walking through the instructions (and maybe resubmit/reopen) but looks good for now!

**Comment by ndushay** [comment]
released as v1.0.1 and deployed to -dev and -stage, just for good measure.
